README.md

```diff
@@ -1,10 +1,47 @@
-## Dock DID Auth
+# Dock Web3 ID
 
-This service provides an oauth solution for authenticating with DIDs through Verifiable Credentials.
+This service provides a SSI-friendly authentication solution for use with DIDs and Verifiable Credentials. There is an existing live service hosted at https://auth.dock.io however you may clone this repository/deploy it to vercel to spin up your own instance. It is mostly stateless, but does rely on a memcached instance currently for short-lived data transfer. No long term storage of user data is used, tokens and codes are obtained through cryptography. You can use this service to allow your users to provide their own user data, like you would request from "Login with Facebook" or "Sign in with Github".
 
-## Getting Started
+Note: You will still want to verify a users email if requested with this service.
 
-First, setup [#env-vars](the environment variables) and pre-requisite services and then you can run the development server:
+Features:
+- No long term storage of user/client data
+- User provides and controls their own data
+- Uses `did:dock` and `did:key` DIDs
+- Cryptographic client id/secrets
+- Oauth2 spec compliant
+
+Roadmap:
+- Support requesting specific credentials/data
+- Decouple verification from the Dock API
+- Look into supporting the SIOP spec
+- OpenID Connect implementation
+- Support more DID types
+- Support non self-signed credentials
+
+## As an oauth2 provider
+
+This service can be used directly as an oauth2 provider with your favourite oauth library. See the documentation for [Oauth2 setup](docs/oauth2_setup.md). You are welcome to use our hosted version or your own. Setup instructions are for the hosted vesion, simply replace with your own domain to configure for another endpoint.
+
+## As an Auth0 connection
```

Comment:
remove Auth0 reference for now

Reply:
i will be removing auth0 in separate PR for easy revert